### DISCUSSION TITLE
مشکل در خصوص پرداخت هایی که موفق بوده ولی به سایت برگشت داده نشدند

### DISCUSSION THREAD

**Discussion by sh008** [Sina-Soltani/Parbad#271] [Q&A]
با سلام و احترام
در خصوص مواردی که کاربر دکمه تکمیل پرداخت رو نمیزنه ویا در هنگام بازگشت اینترنت کاربر به مشکل بر میخوره 
من یک اسکجول دارم که این موارد رو پرداختشون رو چک کنم آیا برای این موارد متدی موجود هست؟
برای مثال در مستندات زرین پال یک متد لیست 
unverify 
وجود داره آیا این مورد داخل این سیستم هستش؟
ممنون

**Comment by Sina-Soltani** [comment]
درود

خیر متدی موجود نیست. در نظر داشته باشید که اگر صورت حساب به هر دلیلی (مشکل اینترنت، برنگشتن به فروشگاه و غیره) توسط وب سایت فروشنده تایید یا به اصطلاح وریفای نشه، اون صورت حساب به صورت خودکار برگشت میخوره و به حساب بانکی خریدار واریز میشه. در نتیجه نیازی به ادامه پیگیری یک پرداخت نیست.

**Reply by sh008** [reply]
درسته ولی متاسفانه سیستم زرین پال چیزی به عنوان بازگشت وجه نداره در هر صورت اگه از سمت سایت هم وریفای نشه باز پول رو واریز میکنه

**Comment by Sina-Soltani** [comment]
بله حق با شماست. به هر حال درگاه‌های واسط به دلیل مصنوعی بودنشون مشکلات فراوانی دارن که یکی از اونها نداشتن امکان بازگشت هزینه هست
